### ISSUE TITLE
I am working with LG WebOS + Shaka player and I wanted to remove src from the video tag when need to play the Multicast channel in WebOS (recommended by LG team ) but it doesn't works

### ISSUE THREAD

**Issue by cbpanchal** [shaka-project/shaka-player#4151]
this.player?.unload()
 this.playerObjectRef?.current.removeAttribute('src')
 this.playerObjectRef?.current.load()
 
 
Written above code but video tag still have a src attribute and not able to play multicast channel after playing a shaka player in LG WebOS TV, and it's recommended to release all video tag instances before playing Multicast channel

**Comment by joeyparrish** [comment]
`unload()` only unloads the content, but keeps control of the video element.  If it still controls the video element, it will set up another MediaSource object in advance for the next `load()` call.  Try this instead:

```js
  // unloads content and gives up its control of the video element (async)
  await this.player.detach();
```

Later, to take control of the video element again:

```js
  await this.player.attach(videoElement);
```

References:
 - https://shaka-player-demo.appspot.com/docs/api/shaka.Player.html#attach
 - https://shaka-player-demo.appspot.com/docs/api/shaka.Player.html#detach
 - https://shaka-player-demo.appspot.com/docs/api/shaka.Player.html#unload

Does this help?

**Comment by cbpanchal** [comment]
@joeyparrish you mean just need to call?

`await this.player.detach();`
 
  instead
  
  `this.player?.unload()`
  
 
 and one more thing is detach will remove the video instance from the dom?

**Comment by theodab** [comment]
Detach un-associates Shaka Player from the video element. It does not remove the video element from the DOM.
